pex/resolve/locker.py

```diff
@@ -309,6 +309,11 @@ def lock_result(self):
 
 
 # See https://www.python.org/dev/peps/pep-0508/#environment-markers for more about these values.
+_OS_NAME = {
+    TargetSystem.LINUX: "posix",
```

Comment:
The list:
+ 3.10: https://docs.python.org/3/library/os.html#os.name
+ 2.7: https://docs.python.org/2.7/library/os.html#os.name

Either way, we only care about "nt" and "posix".